bin/tacl_cl_parser.py

```diff
@@ -407,17 +412,20 @@ def issue_info_to_node(
         volume.append(papernode)
         i += 1
 
-    for paper in volume:
-        for field in paper:
-            field.tail = "\n    "
-        if len(paper):
-            paper.text = "\n    "
-            paper[-1].tail = "\n  "
-        paper.tail = "\n\n  "
-    if len(volume):
-        volume.text = "\n  "
-        volume[-1].tail = "\n"
-    volume.tail = "\n"
-
-    et = etree.ElementTree(volume)
+    for volume in collection:
+        for paper in volume:
+            for field in paper:
+                field.tail = "\n      "
+            if len(paper):
+                paper.text = "\n      "
+                paper[-1].tail = "\n    "
+            paper.tail = "\n\n    "
+        volume.text = "\n    "
+        volume[-1].tail = "\n  "
+    if len(collection):
+        collection.text = "\n  "
+        collection[-1].tail = "\n"
+    collection.tail = "\n"
+
+    et = etree.ElementTree(collection)
     et.write(args.outfile, encoding="UTF-8", xml_declaration=True)
```

Comment:
I usually add `with_tail=True` to the call to `write()`—I don't know why right now, but I think it might be important for getting the indentation right.